docs/tutorial/agent-on-docker.md

```diff
@@ -0,0 +1,157 @@
+      bulk_insert_chunk_size: 10
+      # distance_type, which should be "l1", "l2" "angle", "hamming", "cosine", "normalizedangle" or "nomralizedcosine"
+      distance_type: l2
+      # object_type, which should be "float" or "uint8"
+      object_type: float
+      # creation edge size
+      creation_edge_size: 20
+      # search edge size
+      search_edge_size: 10
+      # in-memory mode enabled
+      enable_in_memory_mode: true
+      # The limit duration of automatic indexing 
+      auto_index_duration_limit: 1m
+      # Check duration of automatic indexing.
+      auto_index_check_duration: 10s
+      # The number of cache to trigger automatic indexing
+      auto_index_length: 100
+    EOF
+    ```
+
+1. Deploy Vald Agent on Docker
+
+    To deploy Vald agent on docker with `config.yaml`, you can run below command.
+
+    ```bash
+    docker run -v $(pwd)/config.yaml:/etc/server/config.yaml -p 8081:8081 --rm -it vdaas/vald-agent-ngt
+    ```
+
+1. Verify
+
+    If the deployment success, you can confirm the output will be similar to below.
+
+    ```bash
+    {"Date":"2020-06-24T05:49:53.615263Z","Level":"INFO","Detail":"maxprocs: Leaving GOMAXPROCS=4: CPU quota undefined"}
+    {"Date":"2020-06-24T05:49:53.615263Z","Level":"INFO","Detail":"service agent ngt v0.0.0 starting..."}
+    {"Date":"2020-06-24T05:49:53.615263Z","Level":"INFO","Detail":["daemon start"]}
+    {"Date":"2020-06-24T05:49:53.620263Z","Level":"INFO","Detail":"server agent-grpc executing preStartFunc"}
+    {"Date":"2020-06-24T05:49:53.620263Z","Level":"INFO","Detail":"gRPC server agent-grpc starting on 0.0.0.0:8081"}
+    ```
+
+## Run using example code
+
+1. Download dataset
+
+    In this tutorial. we use [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
+
+    ```bash
+    # move to working directory
+    cd example/client/agent
+    
+    # download fashion-mnist testing dataset
+    wget http://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+    ```
+
+1. Running example
+
+    Vald provides multiple language client libraries such as Go, Java, Node.js, Python, and so on.<br>
+    In this example, the fashion-mnist dataset will insert into the Vald and search using [vald-client-go](https://github.com/vdaas/vald-client-go).
+    
+    We use [`example/client/agent/main.go`](https://github.com/vdaas/vald/blob/master/example/client/agent/main.go) to run the example.
+    The example code is the same as running an example only Vald agent on Kubernetes.
+    If you want to learn the detail of running an example, please refer to the tutorial of [standalone Vald Agent on kubernetes](../tutorial/get-started.md/#run-using-example-code-1).
+
+    ```bash
+    # run example
+    go run main.go
+    ```
+    Note:
+      - We recommend you to run `CreateIndex()` after `Insert()` without waiting auto indexing.
+
+1. Clean Up
+
+    Stop the Vald Agent docker instance via `Ctrl+C`.
```

Comment:
```suggestion
    Stop the Vald Agent container via `Ctrl+C`.
```

Reply:
I think "the Vald Agent docker container" is the most correctly.
How about it? @kevindiu @hlts2

Reply:
ok!
It seems good to me.

Reply:
It seems good to me too!